components/gitpod-db/src/typeorm/migration/1695735203768-CostCenterDropDeleted.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+export class CostCenterDropDeleted1695735203768 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (await columnExists(queryRunner, "d_b_cost_center", "deleted")) {
+            await queryRunner.query("ALTER TABLE `d_b_cost_center` DROP COLUMN `deleted`");
```

Comment:
do you have a sense of how expensive dropping a column on this table is in prod?

Reply:
DROP is not blocking in any way, especially since it's not indexed. It should be super quick. I can test on a copy of the DB, though, if that helps.

Reply:
Not necessary if you are confident